src/main/java/org/alliancegenome/curation_api/dao/base/BaseSQLDAO.java

```diff
@@ -61,7 +61,9 @@ protected BaseSQLDAO(Class<E> myClass) {
 
 	@Transactional
 	public E persist(E entity) {
-		Log.debug("SqlDAO: persist: " + entity);
+		if(Log.isDebugEnabled()) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.
<!-- __reviewdog__:ChA5M2NkN2JkMWI4NDc3MDMyEgpjaGVja3N0eWxl -->